src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,207 @@
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border button-cell">
+              Modified Date
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  sortColumn(
+                    'MODIFIED_ON' as AccessApprovalSortField,
+                    isDescending,
+                  )
+                }
+              >
+                <IconSvg
+                  options={{
+                    icon: 'sortTwoTone',
+                  }}
+                />
+              </button>
+            </th>
+            <th className="thead-cell remove-border button-cell">
+              Expires
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  sortColumn(
+                    'EXPIRED_ON' as AccessApprovalSortField,
+                    isDescending,
+                  )
+                }
+              >
+                <IconSvg
+                  options={{
+                    icon: 'sortTwoTone',
+                  }}
+                />
+              </button>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {searchResult?.results.map(item => {
+            let modifiedOn = moment(item.modifiedOn).format('L LT')
+            let expiredOn = moment(item.expiredOn).format('L LT')
+            return (
+              <tr key={item.id}>
+                <td className="remove-border">
+                  <a
+                    href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!AccessRequirement:AR_ID=${item.accessRequirementId}`}
+                  >
+                    {item.accessRequirementId}
+                  </a>
+                </td>
+                <td className="remove-border">{item.accessRequirementName}</td>
+                <td className="remove-border">
+                  <UserCard size={SMALL_USER_CARD} ownerId={item.submitterId} />
+                </td>
+                <td className="remove-border">
+                  {item.state[0] + item.state.slice(1).toLocaleLowerCase()}
+                </td>
+                <td className="remove-border">{modifiedOn}</td>
+                <td
+                  className={`${
+                    new Date() > new Date(expiredOn) ? 'expired' : ''
+                  } remove-border`}
+                >
+                  {expiredOn}
+                </td>
+              </tr>
+            )
+          })}
+        </tbody>
+      </Table>
+      {searchResult?.nextPageToken &&
+        (isLoading ? (
+          <SynapseSpinner size={30} />
+        ) : (
+          <button className="load-more-button" onClick={onLoadMore}>
+            Show More
+          </button>
```

Comment:
Switch to react-bootstrap's `Button` and set the `variant` prop to `'outline'`. We've defined this button style in bootstrap already, so hopefully you can get rid of this `load-more-button` class